app/models/work_package/pdf_export/common/logo.rb

```diff
@@ -0,0 +1,23 @@
+module WorkPackage::PDFExport::Common::Logo
+  def logo_image
+    image_obj, image_info = pdf.build_image_object(logo_image_filename)
+    [image_obj, image_info]
+  end
+
+  def logo_image_filename
+    image_file = custom_logo_image_filename
+    image_file = Rails.root.join("app/assets/images/logo_openproject.png") if image_file.nil?
+    image_file
```

Comment:
This could be shortened to: 

```suggestion
    custom_logo_image_filename || Rails.root.join("app/assets/images/logo_openproject.png")
```

Reply:
Thanks! Much nicer.